### PR TITLE
Feedback form on success behavior

### PR DESCRIPTION
**Pull request by virginiacc**
Makes form replacement the default success behavior for feedback forms. Currently some feedback forms are replaced by the success notification (on `owning-a-home/help-us-improve` & Spanish Ask answer pages), but the default behavior (on '/owning-a-home', Ask English answer pages) is to leave the form on the page and add the notification to it. This makes the behavior consistent across the feedback form modules.

## Additions

- Add method to `Notification.js` to get the notification element.

## Changes
- In `feedback.html`
  - Add the block background to the feedback form itself instead of the containing div
  - Move the notification into the feedback form so warning messages appear inside the background
- In `feedback-form.js`
  - Always replace feedback form with success notification
- In `FormSubmit.js`
  - If `replaceForm` option selected, swap form out for its notification module (ensures that notification doesn't appear inside `block__bg`)

## Testing

- Check out branch & run gulp
- Navigate to [Ask answer page](http://localhost:8000/ask-cfpb/how-quickly-can-i-get-money-after-i-deposit-a-check-into-my-checking-account-what-is-a-deposit-hold-en-1023/) and verify that feedback form is replaced on success

## Screenshots
Feedback form error:
screen shot 2017-07-10 at 3 22 29 pm

Feedback form success:
screen shot 2017-07-10 at 3 23 57 pm

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

**Comment by richaagarwal** [comment]
This looks good!  I think this would be well suited for a browser test.  I am happy to pair on it with you if you'd like!

File: cfgov/unprocessed/js/molecules/Notification.js [hunk 1]
```diff
@@ -133,6 +133,10 @@ function Notification( element ) { // eslint-disable-line max-statements, inline
     return this;
   }
 
+  function getElement() {
```
Comment (sebworks) [inline cfgov/unprocessed/js/molecules/Notification.js]: I would get rid of the getter here. I don't think it does anything for you.

Comment (virginiacc) [inline cfgov/unprocessed/js/molecules/Notification.js]: I'm using this for on success form replacement for forms with backgrounds. Error notifications should show inside the background:

screen shot 2017-07-18 at 10 59 20 am
 
but the background isn't necessary when the form gets replaced with the success message:
screen shot 2017-07-18 at 11 15 26 am

so in the FormSubmit form replacement code, I swap out the form for the notification element it contains.

**Comment by virginiacc** [comment]
@richaagarwal Sorry, I missed your comment earlier! I would love to pair on adding a browser test. Can we merge this and work on the test next week, or should they go in together?

File: cfgov/unprocessed/js/molecules/Notification.js (hunk 1 above)
Comment (sebworks) [inline cfgov/unprocessed/js/molecules/Notification.js]: I'm saying why not just expose it with a property like: 

`this.element = _dom`

**Comment by richaagarwal** [comment]
I'd prefer we merge it once the test is in. Technically we shouldn't be merging new code in without test coverage, but I think coveralls only counts python code.

File: cfgov/unprocessed/js/molecules/Notification.js (hunk 1 above)
Comment (virginiacc) [inline cfgov/unprocessed/js/molecules/Notification.js]: Oh, got it. Yep, that would work too!

**Comment by hillaryj** [comment]
Is there a plan to create a test and merge this, or is it ready once it's updated?

**Comment by wpears** [comment]
Subsumed by https://github.com/cfpb/cfgov-refresh/pull/4761

**Pull request closed without merging**